tests/integration/test_ethernet.py

```diff
@@ -0,0 +1,89 @@
+from unittest import mock
```

Comment:
Ideally, the script should also work with Python 2, so we can use it on RHEL 6/7, too.

Reply:
I added an if statement to consider this difference, thanks for pointing it!